locations/dragonkiez-adlerhalle.yml

```diff
@@ -3,6 +3,7 @@ location: dragonkiez-adlerhalle
 location_nice: Dragonkiez Adlerhalle
 latitude: 52.495065429
 longitude: 13.387666941
+altitude: 2
```

Comment:
can't be right

Reply:
This value is from the wiki. Its likely the height. According to google earth the altitude could be 31. Should we go with this value and set the `2` as a `height`?

Reply:
yes, please use 2m as a height and elevation here is 33m so altitude would be 35m